src/Routes/ImageManager/Images.js

```diff
@@ -81,6 +100,106 @@ const Images = () => {
     }),
     shallowEqual
   );
+
+  useEffect(() => {
+    if (data) {
+      setRows(
+        flatten(
+          data.map(item => {
+            const statusIconHash = {
+              "Ready": <CheckCircleIcon color="var(--pf-global--success-color--100)" />,
+              "Image build in progress": <InProgressIcon color="var(--pf-global--palette--blue-400)" />,
+              "Building error": <ExclamationCircleIcon color="var(--pf-global--danger-color--100)" />
+            } 
+            return [
+              {
+                id: item.id,
+                currentStatus: item?.request.status,
+                rowArray: [
+                  item.request.name,
+                  item.request.version,
+                  item.request.distribution,
+                  item.request.type,
+                  item.created_at,
+                  item.request.status,
+                ],
+                cells: [
+                  {
+                    title: (
+                      <Link
+                        to={`${paths['manage-images']}/${item.id}`}
+                      >
+                        {item.request.name}
+                      </Link>
+                    ),
+                  },
+                  item?.request.version,
+                  item?.request.distribution,
+                  item?.request.type,
+                  item?.created_at,
+                  {
+                    title: (
+                      <Flex>
+                        <FlexItem>
+                          {statusIconHash[item?.request.status]}
+                        </FlexItem>
+                        <FlexItem>
+                          {item?.request.status}
+                        </FlexItem>
+                      </Flex>
+                    )
+                  },
+                ],
+              },
+            ];
+          })
+        )
+      )
+    }
+  },[])
+
+  const actionResolver = (rowData) => {
```

Comment:
This looks like a placeholder but can confuse users. Please remove this and we will work on it soon.